mapie/metrics.py

```diff
@@ -737,3 +740,578 @@ def hsic(
     coef_hsic = np.sqrt(np.matrix.trace(hsic_mat, axis1=1, axis2=2))
 
     return coef_hsic
+
+
+def jitter(
```

Comment:
Not sure about the name of this function, would `add_jitter` not be more explicit?

Reply:
Done